CefSharp.Wpf/Rendering/AbstractRenderHandler.cs

```diff
@@ -61,6 +61,7 @@ public bool IsDisposed
         public void Dispose()
         {
             Dispose(true);
```

Comment:
> If obj does not have a finalizer, the call to the SuppressFinalize method has no effect.

https://docs.microsoft.com/en-us/dotnet/api/system.gc.suppressfinalize?view=netframework-4.7.2

So looks like it's a `NO OP`, not sure why it's in the example.

Reply:
It's for when a derived class has a Finalizer